docs/configuration/actions.md

```diff
@@ -65,8 +65,8 @@ These apply to `propose-downstream` command/job and `pull-from-upstream` job.
 
 |        | name                  | working directory | when run                                                              | description                             |
 |--------|-----------------------|-------------------|-----------------------------------------------------------------------|-----------------------------------------|
-| [hook] | `post-upstream-clone` | upstream git repo | after cloning of the upstream repo (main) and before other operations |                                         |
-| [hook] | `pre-sync`            | upstream git repo | after cloning and checkout to the correct (release) branch            |                                         |
+| [hook] | `post-upstream-clone` | upstream git repo | after cloning the upstream repo (main) and dist-git target repo and before gpg keys verification and version compatibility check |                               |
```

Comment:
I see it's really long, but I would add two more pieces of info:
* That we've checked out the correct dist-git branch.
* Link the version compatibility check (e.g. the link to the config option).

Reply:
I wrote *dist-git target repo* meaning that the checkout of the target branch was done. Isn't it understandable? 
Ok for the link to the compatibility check!

Reply:
I read `target repo` as the dist-git repo without any info about what is checked out. Maybe something like this?

```suggestion
| [hook] | `post-upstream-clone` | upstream git repo | after cloning the upstream repo (main) and dist-git (target branch) and before gpg keys verification and version compatibility check |                               |
```